README.md

```diff
@@ -1,6 +1,14 @@
 # Security Belts
 
-## Methodology
+With modern software development methodologies such as DevOps, development teams have the need to take care and also take responsibility for software security of their product themselves. In order to achieve this, the team has to deal with security early on and continuously in its software development process. To empower the teams to handle software security well they have to build up knowledge and awareness for the subject and are willing to continuously improve themselves. But also organizations have to deal with these new requirements and have to enable software security to function decentralized within their organization landscape.
+
+The Security Belts model is defined as a maturity model and thus pursues the idea of giving the teams instructions on how they can best deal with the issue of software security without overwhelming themselves with the complexity that the subject entails. In addition to specifying specific activities to improve software security, we also want to use the model to convey values that are based on [The Five Ideals Of DevOps](https://itrevolution.com/five-ideals-of-devops/) which we believe are essential to successfully develop secure software in the long term. In order to name the individual degrees of maturity, we have based ourselves on the idea of belts, which are known from various martial arts such as judo.
+
+Every activity in this model that is supposed to improve software security is sorted into one of the belts according to its benefit / cost ratio. Activities in the lower belts are not so expensive in terms of time and skill requirements and probably bring the most benefits for software security. The higher the belt, the more demanding the activities and their benefits are less generally applicable.
```

Comment:
```suggestion
Every activity in this model that improves software security is sorted into one of the belts according to its benefit to cost ratio. Activities in the lower belts are not so expensive in terms of time and skill requirements and probably bring the most benefits for software security. The higher the belt, the more demanding the activities and their benefits are less generally applicable.  At the moment benefit and cost values are based on expert judgment. In the future we seek to measure these values, so that the model get's more acurate and profound.
```